src/main/java/io/jenkins/plugins/analysis/warnings/axivion/AxivionSuite.java

```diff
@@ -0,0 +1,240 @@
+package io.jenkins.plugins.analysis.warnings.axivion;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nonnull;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.auth.UsernamePasswordCredentials;
+
+import com.cloudbees.plugins.credentials.CredentialsMatchers;
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import com.cloudbees.plugins.credentials.common.StandardListBoxModel;
+import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+
+import edu.hm.hafner.analysis.ParsingCanceledException;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.Report;
+
+import org.kohsuke.stapler.AncestorInPath;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.model.Item;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.security.ACL;
+import hudson.util.FormValidation;
+import hudson.util.ListBoxModel;
+import hudson.util.Secret;
+import jenkins.model.Jenkins;
+
+import io.jenkins.plugins.analysis.core.model.Tool;
+import io.jenkins.plugins.analysis.core.util.EnvironmentResolver;
+import io.jenkins.plugins.analysis.core.util.LogHandler;
+
+/** Provides a parser and customized messages for the Axivion Suite. */
+public class AxivionSuite extends Tool {
+
+    private static final long serialVersionUID = 8670135727302169818L;
+    static final String ID = "AxivionStoppingSoftwareErosion";
```

Comment:
quite a long URL, is this intended?

Reply:
Yep, it was intented, but AxivionSuite is ok too. Changed it.

Reply:
It is used as URL to the results, so typically it does contain only lowercase characters (separated by -), see https://github.com/jenkinsci/warnings-ng-plugin/blob/master/SUPPORTED-FORMATS.md.